core/app/models/spree/order.rb

```diff
@@ -400,7 +400,7 @@ def deliver_order_confirmation_email
 
     # Helper methods for checkout steps
     def paid?
-      payment_state == 'paid' || payment_state == 'credit_owed'
+      %w(paid credit_owed).include?(payment_state)
```

Comment:
Aye, I appreciate its more accurate but personally would rather leave it as an array as it is everywhere else in the codebase and then deal with the set on a global level rather than having the two mixed uses of convention.